src/commands/Management/createMute.ts

```diff
@@ -26,6 +28,7 @@ export default class extends SkyraCommand {
 			await message.guild!.settings.update(GuildSettings.Roles.Muted, role, {
 				extraContext: { author: message.author.id }
 			});
+			await message.react(resolveEmoji(Emojis.GreenTick)!);
```

Comment:
I don't think you have to resolve the emoji, as GreenTick is already a valid emoji

Reply:
For texts? Yes.
For reacts? No.

Reply:
[EmojiResolvables](https://discord.js.org/#/docs/main/stable/typedef/EmojiIdentifierResolvable) need IDs or Emoji objects for custom emotes.